django/core/exceptions.py

```diff
@@ -192,3 +192,8 @@ def __repr__(self):
 class EmptyResultSet(Exception):
     """A database query predicate is impossible."""
     pass
+
+
+class NoSuchShell(Exception):
```

Comment:
I don't know if this needs to be a core exception, it seems too specific to the `shell` command to me